bindings/jupyter-modules/jupyter-ma-causal/src/components/slidebar/index.js

```diff
@@ -14,6 +14,7 @@ export default class Chart extends Component {
     onDragStart: PropTypes.func.isRequired,
     onDrag: PropTypes.func.isRequired,
     onDragEnd: PropTypes.func.isRequired,
+    disableDrag: PropTypes.bool.isRequired,
```

Comment:
no need for the `isRequired` as we have it in the `defaultProps`?

Reply:
I will leave them here for now. I will fix all of them later.